pkg/envoy/proxy.go

```diff
@@ -117,6 +117,8 @@ func (e *envoy) args(fname string, epoch int, bootstrapConfig string) []string {
 		"--service-node", e.Node,
 		"--local-address-ip-version", proxyLocalAddressType,
 		"--bootstrap-version", "3",
+		"--use-dynamic-base-id", // Ensure we don't conflict with other Envoys in the same process namespace
```

Comment:
I thought baseid is only needed with hot restart when allocating shared memory https://www.envoyproxy.io/docs/envoy/latest/operations/cli#cmdoption-use-dynamic-base-id - When we are disabling it , do we need this flag?